src/mod_mam_muc_cache_user.erl

```diff
@@ -75,6 +75,9 @@ writer_child_spec() ->
      [?MODULE]}.
 
 start_server(_Host) ->
+    %% TODO make per host server to be consistent with the way
```

Comment:
Can we introduce it in this PR? It doesn't seem to be big change and its one "TODO" less :D

Reply:
nope, should be a separate PR to avoid making a giant change ;)